roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -41,10 +41,6 @@ oom_score = {{ containerd_oom_score }}
 {% if kata_containers_enabled %}
         [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata-qemu]
           runtime_type = "io.containerd.kata-qemu.v2"
-{% endif %}
-{% if gvisor_enabled %}
-        [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runsc]
-          runtime_type = "io.containerd.runsc.v1"
```

Comment:
Why the Configs be removed?
Shoud it be keeped ?

Follow by:
https://gvisor.dev/docs/user_guide/containerd/quick_start/
https://github.com/containerd/containerd/blob/main/docs/cri/config.md?plain=1#L61